types/marked/index.d.ts

```diff
@@ -109,8 +109,8 @@ declare namespace marked {
         codespan(code: string): string;
         br(): string;
         del(text: string): string;
-        link(href: string, title: string, text: string): string;
-        image(href: string, title: string, text: string): string;
+        link(href: string, title: string | null, text: string): string;
+        image(href: string, title: string | null, text: string): string;
```

Comment:
Here, `href` can definitely be nullable, but not sure we should allow `null` in terms of `title` here.

Reply:
@worr
We can get `null` when we pass a markdown like

```md
[This is text](http://example.com)
```

If we explicitly pass the title as follows, it will not be null.

```md
[This is text](http://example.com "This is title")
```